### PR TITLE
Add built-in monitoring configs

### PR DESCRIPTION
**Pull request by PettitWesley**
Signed-off-by: Wesley Pettit <wppttt@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/fluent-bit/health-check/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Please note, health checks are not the only option for monitoring Fluent Bit. Fl
...
 
-See the [task-definition-output-metrics-healthcheck.json](task-definition-output-metrics-healthcheck.json) in this directory. This health check requires a custom config file, to understand how to use custom configuration files please see the base [config-file-type](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/config-file-type-file) example. 
+See the [task-definition-output-metrics-healthcheck.json](task-definition-output-metrics-healthcheck.json) in this directory. This health check uses the built-in `/fluent-bit/configs/output-metrics-healthcheck.conf` configuration available in AWS for Fluent Bit 2.X.Y+. This means you do not need to build a custom image to use this health check. If you still want to use a custom image built with your custom configuration, the `output-metrics-healthcheck.conf` shows the configuration required and the [config-file-type](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/config-file-type-file) example in this guide shows how to use a custom config file.
```
Comment (PettitWesley) [inline examples/fluent-bit/health-check/README.md]: Must wait till we merge and release: https://github.com/aws/aws-for-fluent-bit/pull/485

File: examples/fluent-bit/send-fb-internal-metrics-to-cw/README.md [hunk 2]
```diff
@@ -50,6 +50,20 @@ This example contains the following:
 
 This is necessary because currently, the [Fluent Bit metrics](https://docs.fluentbit.io/manual/pipeline/inputs/fluentbit-metrics) and the [Prometheus Metrics](https://docs.fluentbit.io/manual/pipeline/inputs/prometheus-scrape-metrics) inputs do not emit their data as logs, and use a separate metric pipeline that most Fluent Bit plugins do not support. However, if we use the exec input to curl the metrics, then the outputted prometheus data is turned into logs which we can parse and process easily. In the future, this experience may be improved.
 
+For a quick setup, use the built-in plugin metrics configuration file available in AWS for Fluent Bit 2.X.Y+. This built-in configuration means that you do not have to build a custom Fluent Bit image. However, please note that this built-in configuration enables all plugin metrics (not just output plugin metrics).
```
Comment (PettitWesley) [inline examples/fluent-bit/send-fb-internal-metrics-to-cw/README.md]: Must wait till we merge and release: https://github.com/aws/aws-for-fluent-bit/pull/485

File: examples/fluent-bit/health-check/README.md (hunk 1 above)
Comment (matthewfala) [inline examples/fluent-bit/health-check/README.md]: This link doesn't work

**Review by matthewfala — CHANGES_REQUESTED** [review]
There's a broken link

File: examples/fluent-bit/health-check/README.md (hunk 1 above)
Comment (PettitWesley) [reply examples/fluent-bit/health-check/README.md]: This is merged but not yet released. Should be part of 2.30.0

Comment (PettitWesley) [reply examples/fluent-bit/health-check/README.md]: We just released this in 2.29.1

**Review by matthewfala — APPROVED** [review]
This is great!

**Review by Claych — APPROVED** [review]

**Pull request merged**